store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/MiniAccumuloStore.java

```diff
@@ -148,11 +148,15 @@ private void createCluster() throws IOException, InterruptedException {
 
         MiniAccumuloConfig config = new MiniAccumuloConfig(getAccumuloDirectory(), rootUserPassword);
 
-        String[] zookeepers = getProperties().getZookeepers().split(":");
-        if (zookeepers.length == 2) {
-            config.setZooKeeperPort(Integer.parseInt(zookeepers[1]));
-        } else {
+        if (getProperties().getZookeepers() == null) {
             config.setZooKeeperPort(DEFAULT_ZOOKEEPER_PORT);
+        } else {
+            String[] zookeepers = getProperties().getZookeepers().split(":");
+            if (zookeepers.length == 2) {
+                config.setZooKeeperPort(Integer.parseInt(zookeepers[1]));
+            } else {
+                config.setZooKeeperPort(DEFAULT_ZOOKEEPER_PORT);
```

Comment:
This could be refactored such that the `config.setZooKeeperPort(DEFAULT_ZOOKEEPER_PORT)` call occurs only once.

Reply:
Done.